### PR TITLE
Check-in page shows email, phone, and name

### PR DESCRIPTION
**Pull request by webnard**
@alexcdahl Do you mind taking a look at this? Thanks!

### CODE REVIEW

File: core/app_modules/user.js [hunk 1]
```diff
@@ -51,5 +53,79 @@ module.exports = {
...
+  ldapInfo: co.wrap(function*(netid) {
+    var data = {};
+
+    if(!arguments.length) {
+      throw new Error("No arguments passed.");
+    }
+
+    for(let i = 0; i<arguments.length; i++) {
+      if(arguments[i].match(/[^a-z0-9]/i)) {
+        throw new Error("Invalid NetID: " + arguments[i])
```
Comment (alexcdahl) [inline core/app_modules/user.js]: It looks good, but I feel like I haven't adequately reviewed if it I don't make a comment, so I'm nitpicking. You missed a semicolon here.

**Comment by alexcdahl** [comment]
This looks good to me. You may merge when ready.

File: core/app_modules/user.js (hunk 1 above)
Comment (webnard) [inline core/app_modules/user.js]: Thanks. I'll add one in.

**Comment by webnard** [comment]
@alexcdahl There, semicolons and an extra layer of abstraction. How does it look now?

**Comment by alexcdahl** [comment]
Looks good. And it passed the build. https://www.youtube.com/watch?v=uvoUvDKh9SA

**Comment by webnard** [comment]
Roger.

**Pull request closed without merging**